File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikInstructionsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.instructions.StepikInstructionsGetQuery;

/**
 * @author meanmail
 */
public class StepikInstructionsAction extends StepikAbstractAction {
    public StepikInstructionsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikInstructionsGetQuery get() {
        return new StepikInstructionsGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikInstructionsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.instructions.StepikInstructionsGetQuery

class StepikInstructionsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikInstructionsGetQuery {
        return StepikInstructionsGetQuery(this)
    }
}
